onnxruntime/core/providers/rocm/tunable/gemm_hipblaslt.h

```diff
@@ -19,6 +19,12 @@ namespace internal {
 
 #ifdef USE_HIPBLASLT
 
+typedef enum _ActivationType {
```

Comment:
In C++, we don't do this, just
```cpp
enum ActivationType { ... };
```
is good to go. This is a pattern in C but not used in C++